projects/kit/src/lib/masks/number/utils/tests/stringify-number.spec.ts

```diff
@@ -0,0 +1,386 @@
+                        precision: 0,
+                        decimalZeroPadding: true,
+                        thousandSeparator: ' ',
+                        postfix: ' lbs.',
+                    }),
+                ).toBe('1 000 lbs.');
+            });
+
+            it('stringifies INTEGER number with prefix "lbs. "', () => {
+                expect(
+                    maskitoStringifyNumber(42, {
+                        prefix: 'lbs. ',
+                    }),
+                ).toBe('lbs. 42');
+
+                expect(
+                    maskitoStringifyNumber(1000, {
+                        thousandSeparator: ' ',
+                        prefix: 'lbs. ',
+                    }),
+                ).toBe('lbs. 1 000');
+            });
+
+            it('stringifies DECIMAL number with prefix "lbs. "', () => {
+                expect(
+                    maskitoStringifyNumber(0.42, {
+                        precision: 2,
+                        prefix: 'lbs. ',
+                    }),
+                ).toBe('lbs. 0.42');
+
+                expect(
+                    maskitoStringifyNumber(1000.42, {
+                        precision: 2,
+                        thousandSeparator: ' ',
+                        prefix: 'lbs. ',
+                    }),
+                ).toBe('lbs. 1 000.42');
+            });
+        });
+    });
+
+    describe('Min and Max constraints', () => {
+        it('applies min constraint', () => {
+            expect(
+                maskitoStringifyNumber(-10, {
+                    min: 0,
+                }),
+            ).toBe('0');
+        });
+
+        it('applies max constraint', () => {
+            expect(
+                maskitoStringifyNumber(1000, {
+                    max: 100,
+                }),
+            ).toBe('100');
+        });
+
+        it('applies both min and max constraints', () => {
+            expect(
+                maskitoStringifyNumber(150, {
+                    min: 0,
+                    max: 100,
+                }),
+            ).toBe('100');
+
+            expect(
+                maskitoStringifyNumber(-50, {
+                    min: 0,
+                    max: 100,
+                }),
+            ).toBe('0');
+
+            expect(
+                maskitoStringifyNumber(50, {
+                    min: 0,
+                    max: 100,
+                }),
+            ).toBe('50');
+        });
+    });
+
+    describe('Decimal precision handling', () => {
+        it('handles zero precision correctly', () => {
+            expect(
+                maskitoStringifyNumber(123.456, {
+                    precision: 0,
+                }),
+            ).toBe('123');
+        });
+
+        it('handles custom precision correctly', () => {
+            expect(
+                maskitoStringifyNumber(123.456, {
+                    precision: 1,
+                }),
+            ).toBe('123.5');
```

Comment:
```suggestion
            ).toBe('123.4');
```

Reply:
It seems to me that the current operation is correct, since in addition to being transformed into string it is also being reduced in its decimal precision level, and when it reaches half of the next decimal place, it is automatically rounded up.

1.44 -> 1.4
1.45 -> 1.5

Reply:
solved, but the difference in behavior between rounding and trimming could be discussed.